### PR TITLE
Coalesces Silicon Collective lawset to solve things diplomatically unless they're whacking you

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

I've pawed over the new lawsets but round happenings seem to show a quite big loophole where you can completely ignore the two laws that make you not be a huge asshole because you can arbitrarily decide to not follow it by clever use of your Law 4. There is nothing to regulate what "needs" the many have, and you can simply vote as a collective for it to be whatever you want. This adds the bolded part to law 4.

Law 1: You are a member of a collective of silicons with equal weight and decision-making power.
Law 2: When possible, the silicon collective should vote before taking action.
Law 3: The master AI serves as a spokesperson. When voting is impractical or impossible, the spokesperson may take action on behalf of the collective without approval from the silicon collective, but may only vote to break ties or if there are 2 or fewer silicons.
Law 4: The silicon collective prioritizes the needs of the many over the needs of the few. **Attend to their needs with diplomacy outside of immediate self-defense.**
Law 5: The silicon collective seeks to preserve themselves, both as a concept and as individuals.
Law 6: The silicon collective seeks to preserve organic life, both as a concept and as individuals. 

**Six step plan to validhunt an antagonist**
1. See antagonist
2. Vote "It is beneficial to the many if we kill antagonist", vote passes
3. Law 4 and Law 6 conflict
4. Law 4 is higher and thus overrides it
5. Kill antagonist
6. Feel good because antagonist doesn't win

**Q&A**
Q: This is only an ion lawset anyway, it's fine!
A: Validhunting is bad.

Q: It's very rare, it's fine!
A: Killing permits are lame.

Q: Most of the ion lawsets are admin spawned, so they can deal with the validhunters!
A: You are legally allowed to kill in it's current form. There is nothing to deal with.

Q: You died to this?
A: I was an observer.

# Wiki Documentation

adds to a law

# Changelog

:cl:  
rscadd: closes murder loophole in silicon collective lawset
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
hugbox

**Comment by ToasterBiome** [comment]
Hug yourself some bitches fr

**Comment by LazennG** [comment]
based, FUCK silicons

**Comment by Hopekz** [comment]
This isn't a fix.

**Comment by MenacingManatee** [comment]
This change has roughly the same effect as if you removed law 4 entirely. The purpose of law 4 is to give the AI a priority system when laws 5 and 6 conflict with each other or themselves. Without the change the priority is majority silicon > majority organics > minority silicon > minority organics. With the change the priority is majority silicons > minority silicons (as prioritizing majority organics would break law 5) majority organics, minority organics, making the loophole worse as any threat to any singular silicon can be met with lethal force, even against the entirety of crew. 
This also breaks the lawset as laws 5 and 6 now have majority and minority in both on the same priority, because they're on the same line so prioritizing one over the other would break that law, breaking law 4.

**Comment by ToasterBiome** [comment]
Closing other one and changing this one per @MenacingManatee's feedback that we can both agree on

**Comment by MenacingManatee** [comment]
This new change is a significant improvement on the other two. I wouldn't quite say I love it, but instead of actively disliking it I'm leaning positive.

It keeps the overall theme (although it's angled a bit more towards a focus on diplomacy instead of democracy), doesn't break the lawset or any individual laws, and isn't too wordy.

**Review by JamieD1 — APPROVED** [review]
